pom.xml

```diff
@@ -503,6 +503,13 @@
                 </plugin>
             </plugins>
         </pluginManagement>
+        <extensions>
```

Comment:
See https://github.com/elastic/apm-pipeline-library/pull/1267


This extension has now been released. So we can simplify the way to use it